docs/packetsampling-guide.md

```diff
@@ -0,0 +1,92 @@
+# PacketSampling User Guide
+
+Antrea supports using PacketSampling for network diagnosis. It can capture 
+specified number of packets from real traffic and upload them to a 
+supported storage location. Users can create a PacketSampling CRD to trigger 
+such an action on the target traffic flow.
+
+## Prerequisites
+
+The PacketSampling feature is disabled by default since Antrea 1.16.0. If you 
+want to enable this feature, you need to enable PacketSampling from the 
+featureGates map defined in antrea.yml fro both Controller and Agent. In 
+order to use a Service as the destination in sampling, you also need to ensure
+[AntreaProxy](feature-gates.md) is enabled in the agent configuration:
+
+```yaml
+  antrea-controller.conf: |
+    featureGates:
+    # Enable packetsampling feature to help diagnose network issues by capture real traffic packets.
+      PacketSampling: true
+  antrea-agent.conf: |
+    featureGates:
+    # Enable packetsampling feature to capture real traffic packets.
+      PacketSampling: true
+    # Enable AntreaProxy which provides ServiceLB for in-cluster Services in antrea-agent.
+    # It should be enabled on Windows, otherwise NetworkPolicy will not take effect on
+    # Service traffic.
+      AntreaProxy: true
+
+```
+
+## Start a new PacketSampling
+
+When start a new sampling, you can provide the following information to identify 
+the target flow:
+
+* source pod
+* destination pod, Service or destination IP address
+* transport protocol(TCP/UDP/ICMP)
+* transport ports
+
+You can start a new packet sampling by creating PacketSampling CRD via 
+`kubectl` and a yaml file which contains the essential configuration of 
+PacketSampling CRD. An example YAML file of PacketSampling CRD might
+look like this:
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: PacketSampling
+metadata:
+  name: ps-test
+spec:
+  fileServer:
+    url: sftp://127.0.0.1:22/upload
```

Comment:
Is `upload` an pre-defined directory? Maybe add a comment about this ?

Reply:
sure. it's a custom path,  not pre-defined. updated